src/main/java/frc/robot/subsystems/hang/docs/Hang.md

```diff
@@ -0,0 +1,58 @@
+|setManualVelocity|void|velocity|sets the velocity of the motor that we decide| |
+|getHangPosition|double|N/A|finds the positon of the hang motor|through rotations|
+|setTargetPosition|void|targetPosition|sets the postion we that want the motor to get to/be at| |
+|isAtPosition|void|N/A|determines if the motor is at the needed poistion| |
+|getError|double|N/A|determines how far off the motor is from where it should be|target position-current position|
+|getHangVelocity|double|N/A|finds the velocity of the motor| |
+|getMotorCurrent|double|N/A|find the currunent being outputed in the motor|in amps|
+|setHangMode|void|HangControlType type|figures out the hang mode to determine how the motor will move|uses a enum class. the modes either run at a constant voltage (MANUAL mode) to test how the arm runs structualy, or the motor goes to a postion using PID (PID mode) for actual use within competition |
+
+## What is Logged To Dashboard
+| name | Description | Notes |
+| --- | --- | ---- |
+|kMotorInverted|logs if the motor is inverted|under setupMotors method|
+|kHangMotorLimit|logs the positon limit of the motor|under setupMotors method|
+|kPositionConversionFactor|logs the conversion factor of the motor position for the encoders|under setupMotors method|
+|kVelocityConversionFactor|logs the conversion factor of the motor velocity for the encoders|under setupMotors method|
+|kBrake|logs if the motors are braked(wont move) for idle mode|under setupMotors method|
+|Kp, Ki, Kd| logs the p,i,and d for motor soothness|under setupMotors method|
+| --- | --- | ---- |
+|Hang Current Velocity|logs the velocity of the motor|under doSendables method|
+|Hang Current Position|logs the position of the motor|under doSendables method|
+|Hang Target Position|logs the postion that is wanted for the motor to go to|under doSendables method|
+|Hang Error|logs the error of the motor (distance from wanted position)|under doSendables method|
+|Hang Current|logs the current of the motor|under doSendables method|
+|Hang Bus Voltage|logs the voltage of the canbus|under doSendables method|
+|Hang Output|logs the power output of the spark max|under doSendables method|
+| --- | --- | ---- |
+|kHangTolerance|logs the tolerance (the allowed error) of the motor|under isAtPosition method|
+
+## How To Use
+to use the subsytem, you need to create a mountable motor connected to a sparkmax/encoder
+you would also need to have the frc dashboard to be able to tweak and modify code for desired function.
+### manual mode (why and how to use)
+manual mode is important to testing the hang mechanism, specifically how stable, and sturcturaly sound the hang is.
```

Comment:
make sure to use backticks (`) to format them as code. You can surround with one backtick for in-line and three for multiline.